modules/sdk/project-templates/project-templates/src/main/java/com/liferay/project/templates/internal/util/FileUtil.java

```diff
@@ -299,4 +271,104 @@ public static void setPosixFilePermissions(
 		}
 	}
 
+	private static Map<String, InputStream> _getFilesFromClasspath(
+			String directoryName)
```

Comment:
A `pathString` or `dirName` is passed into `_getFilesFromClasspath`.

Can we rename the parameter from `directoryName` to `dirPathString`, remove `L138-L141`, and just pass `"/" + dirName` on `L143`?